test/Concurrency/isolated_parameters.swift

```diff
@@ -319,3 +319,32 @@ func isolatedClosures() {
     a.f()
   }
 }
+
+// Test case for https://github.com/apple/swift/issues/62568
+func execute<ActorType: Actor>(
+  on isolatedActor: isolated ActorType,
+  task: @escaping @Sendable (isolated ActorType) -> Void)
+{
+  // Compiler correctly allows this task to execute synchronously.
+  task(isolatedActor)
```

Comment:
a bit confusing to call this "task" but code is good 👍

Reply:
Fair, but I copied the test case from https://github.com/apple/swift/issues/62568